src/utils/index.js

```diff
@@ -1,3 +1,5 @@
 const filterList = (list, value) => list.filter(item => item.name.includes(value));
+const findIndexById = (list, currId) => list.findIndex(item => item.id === currId);
 
 export { filterList };
```

Comment:
I think you can just `export filterList`, same below

Reply:
This syntax doesn't support. Leave as is.